src/altinn-app-frontend/src/components/advanced/AddressComponent.tsx

```diff
@@ -326,12 +325,9 @@ export function AddressComponent({
             readOnly={readOnly}
             labelSettings={labelSettings}
           />
-          <input
+          <TextField
             id={`address_care_of_${id}`}
-            className={cn('form-control', {
-              'validation-error': allValidations.careOf.errors.length,
-              disabled: readOnly,
-            })}
+            isValid={allValidations.careOf.errors.length === 0}
```

Comment:
Did we lose the `readOnly` property in the process of rewriting this?

Reply:
Its still there, just a few lines down (not modified) 🙂

Its just the classname that was removed, which is correct.

Reply:
Haha, I didn't notice there were non-modified lines! Thanks! :see_no_evil: